packages/logging-winston/src/index.js

```diff
@@ -169,7 +169,28 @@ LoggingWinston.prototype.log = function(levelName, msg, metadata, callback) {
     labels: labels
   };
 
-  var entry = this.log_.entry(entryMetadata, msg);
+  // Stackdriver Logs Viewer picks up the summary line from the `message`
+  // property of the jsonPayload.
+  // https://cloud.google.com/logging/docs/view/logs_viewer_v2#expanding.
+  //
+  // For error messages at severity 'error' and higher, Stackdriver
+  // Error Reporting will pick up error messages if the full stack trace is
+  // included in the textPayload or the message property of the jsonPayload.
+  // https://cloud.google.com/error-reporting/docs/formatting-error-messages
+  // We prefer to format messages as jsonPayload (by putting it as a message
+  // property on an object) as that works is accepted by Error Reporting in
+  // for more resource types.
+  //
+  // TODO(ofrobots): when resource.type is 'global' we need to additionally
+  // provide serviceContext.service as part of the entry for Error Reporting to
+  // automatically pick up the error.
+  var data = {
```

Comment:
I believe the behavior of appending a stack trace (both with and without the `: ` prefix) is not being unit tested.